### ISSUE TITLE
Synchronize heat-up

### ISSUE THREAD

**Issue by tocguy** [marian42/octoprint-preheat#56]
Since I've adopted a glass bed its heat-up time takes longer than before; upwards of 5 minutes now to reach 90°C. The nozzle, on the other hand, can reach 240°C in just 2 minutes. PID calibrations have been performed on both so my question is this, can this data be used to synchronize the two heating elements such that the nozzle isn't sitting their oozing for an extended period whilst waiting for the bed to catch-up?

**Comment by marian42** [comment]
There is the Sequenced preheat feature where it will wait until the bed is done before starting to heat the nozzle.

But I understand you're asking for a feature where it learns the duration of the bed and nozzle heating and guesses a start time difference so that they are ready at the same time.

I think this is certainly possible, but I'm afraid it is out of scope for this project.
